PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -673,9 +672,14 @@ struct HfCandidateCreatorXic0Omegac0Mc {
     for (const DeviceSpec& device : workflows.devices) {
       if (device.name.compare("hf-candidate-creator-xic0-omegac0") == 0) {
         for (const auto& option : device.options) {
-          if (option.name.compare("hfEvSel.zPvPosMax") == 0) {
-            zPvPosMax = option.defaultValue.get<float>();
-            break;
+          if (option.name.compare("hfEvSel.useSel8Trigger") == 0) {
```

Comment:
Are you referring to `useItsRoBorderCut` being assigned the value of `useSel8Trigger`? Because the rest is matching?

Reply:
Yes

Reply:
Are you suggesting to drop `useSel8Trigger` completely and have switches for all its subsets?

Reply:
Hi @fcatalan92 . Here the idea is to take the same configuration used at the reco level to avoid mistakes. I know that this might be misleading, but at the reco level at the moment 3 selections are entangled in the sel8.
I would not add additional configurables in the gen particles event selection. I would read also the TVX trigger that you are suggesting to add from this `useSel8Trigger`, at least for the moment. This is the only way to ensure that the same cuts are applied at the two levels. 
The cleaner thing to do at some point would be to disentangle the sel8 in the reco part, but for the moment I would not add additional configurables that might be even more confusing for the analyzer

Reply:
@vkucera I would not drop `useSel8Trigger` but I would add switches also for his components: `kIsTriggerTVX`, `kNoITSROFrameBorder`, `kNoTimeFrameBorder`. Since for the data we want to apply `sel8`, and being able to test the components independently is also useful. While for the MC, we want to apply `kNoTimeFrameBorder` and maybe `kIsTriggerTVX` (to be investigated).

@DelloStritto Since the PR for the reco part is now merged https://github.com/AliceO2Group/O2Physics/pull/6311, I would do the modification to disentangle the `sel8` in the reco part here, to have a consistent approach.
We need to avoid to have the same configurable that does different things on reco and gen, since this can be understood only from reading the code and it is prone to errors.